### PR TITLE
Add support for ContentLanguage and StorageClass

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: src/Network/Minio.hs [hunk 1]
```diff
@@ -102,7 +102,9 @@ module Network.Minio
   , pooContentType
   , pooContentEncoding
   , pooContentDisposition
+  , pooContentLanguage
```
Comment (krisis) [inline src/Network/Minio.hs]: need to add `2018` to copyright header

File: src/Network/Minio/Data.hs [hunk 2]
```diff
@@ -189,14 +189,16 @@ data PutObjectOptions = PutObjectOptions {
     pooContentType :: Maybe Text
   , pooContentEncoding :: Maybe Text
   , pooContentDisposition :: Maybe Text
+  , pooContentLanguage :: Maybe Text
```
Comment (krisis) [inline src/Network/Minio/Data.hs]: need to add `2018` to copyright header

File: test/LiveServer.hs [hunk 3]
```diff
@@ -339,7 +339,76 @@ liveServerUnitTests = testGroup "Unit tests against a live server"
...
+      fPutObject bucket object inputFile def{
+        pooContentLanguage = Just "en-US"
+        }
+
+      -- retrieve obj info to check
+      oi <- headObject bucket object
+      let m = oiMetadata oi
+
+      step "Validate content-language"
+      liftIO $ assertEqual "content-language did not match" (Just "en-US") (Map.lookup "content-language" m)
```
Comment (krisis) [inline test/LiveServer.hs]: It should be `Map.lookup "Content-Language" m`.

File: test/LiveServer.hs [hunk 4]
```diff
@@ -339,7 +339,76 @@ liveServerUnitTests = testGroup "Unit tests against a live server"
...
+      inputFile'' <- mkRandFile size0
+
+      fPutObject bucket object inputFile def{
+        pooStorageClass = Just "STANDARD"
+        }
+
+      fPutObject bucket object' inputFile' def{
+        pooStorageClass = Just "REDUCED_REDUNDANCY"
+        }
+
```
Comment (krisis) [inline test/LiveServer.hs]: Server won't return "STANDARD" storage class in response header during `headObject`, so either we can modify the test or remove it and retain only "REDUCED_REDUNDANCY" test.

Comment (harshavardhana) [reply test/LiveServer.hs]: fixed

File: src/Network/Minio.hs (hunk 1 above)
Comment (harshavardhana) [reply src/Network/Minio.hs]: fixed

**Pull request merged**